File: solutions/cpp/pangram/1/pangram.cpp
#include "pangram.h"
#include <algorithm>
#include <cctype>
#include <string>
#include "algorithm"


namespace pangram {
bool is_pangram(std::string test)
{
    std::transform(test.begin(),test.end(),test.begin(),[](unsigned char c){return std::tolower(c);});
    for(char i='a'; i<='z'; i++)
    {
        if(!test.find(i))
            break;
    }
    return false;
}
}  // namespace pangram

File: solutions/go/pangram/1/pangram.go
package pangram

import "strings"

func IsPangram(input string) bool {
	input = strings.ToLower(input)
	for i := 'a'; i <= 'z'; i++ {
		old := len(input)
		input = strings.Replace(input, string(i), "", 1)
		if len(input) == old {
			return false
		}
	}
	return true
}
